include/fmt/color.h

```diff
@@ -231,6 +235,7 @@ struct color_type {
 }  // namespace detail
 
 /** A text style consisting of foreground and background colors and emphasis. */
+FMT_MODULE_EXPORT
```

Comment:
Why not include this class in the `FMT_MODULE_EXPORT_BEGIN/FMT_MODULE_EXPORT_END` block below and remove `FMT_MODULE_EXPORT`?

Reply:
Yes, why not? I'll tell ya: staring at code for extended periods of time didn't prevent me from being silly.